redbot/cogs/trivia/data/lists/technology.yaml

```
@@ -0,0 +1,205 @@
+- Mars
+What do we call a piece of software that "tells" an operating system how to communicate with the computer's hardware?:
+- A device driver
+- Device driver
+The first copyleft license was made available by GNU (GNU's Not UNIX). This is often called GNU GPL or just GPL. What does GPL stand for?:
+- General Public License
+Which operating system, originally developed by Linus Torvalds, was licensed under the copyleft GNU GPL?:
+- Linux
+What does CSS actually stand for?:
+- Cascading Style Sheets
+What keyword is given to a rule to prevent other conflicting rules from overriding it?:
+- !important
+When elements are positioned absolutely, they can be layered on top of one another. What property determines the stacking order for layered elements?:
+- Z-index
+- Z index
+The process of building an object on the computer within 3D space is most often called what?:
+- Modeling
+Once the character or object is built in the program, what is the process called of adding color, reflection, transparency, translucency and roughness?:
+- Texturing
+Basic shapes, such as cubes, cylinders and circles that are used to build an object are called what?:
+- Primitives
+A collection of computers linked together to render different frames of a single animation are referred to as what?:
+- Render farm
+How big is the standard CD?:
+- 12 cm
+- twelve centimeters
+- twelve cm
+- 12 centimeters
+When they were first introduced, how many minutes of music could be stored on a CD?:
+- 74
+- Seventy four
+What is used to read the data off the data track of a CD?:
+- Laser
+In 1983, this person was the first to offer a definition of the term 'computer virus'.:
+- Cohen
+- Fred Cohen
+- Fred
+A computer virus that actively attacks an anti-virus program or programs in an effort to prevent detection.:
+- Retrovirus
+In what year did the Symantec Corporation first release Norton Antivirus?:
+- 1990
+In 1988 one of the most common viruses was unleashed. Activated every Friday the 13th, the virus affects both .EXE and .COM files and deletes any programs run on that day. What is the name of that virus?:
+- Jerusalem
+Before a Cascading Style Sheet is useful, you need to include one into your HTML page. What HTML tag can you use to apply CSS rules to a document?:
+- Style
+Which SQL command would you use to retrieve a record or records from the database?:
+- SELECT
+- Select
+Which SQL command would you use to add a new record (row) to the database?:
+- INSERT
+- Insert
+Which SQL command would you use to modify the contents of an existing record (row) in the database?:
+- UPDATE
+- Update
+Since there are only 10 different digits, we can't have any base greater than 10.:
+- False
+- No
+What would be the decimal value of 1111 1110?:
+- 254
+A double is classified as what data type?:
+- A floating point data type
+- Floating point data type
+What is the symbol for the modulus operator?:
+- %
```

```suggestion
- "%"
```
This is currently invalid YAML, and fails the tests.